docs/release-source/release/migrating-to-3.0.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Migrating to v3.0 - Sinon.JS
+breadcrumb: migrating to 3.0
+---
+
+Sinon v3.0 is the third major release, we have made several breaking changes in this release as a result of modernising the internals of Sinon.  This guide is intended to walk you through the changes.
+
+## `sinon.stub(object, "method", func)` - Removed
+Please use `sinon.stub(obj, "method").callsFake(func)` instead.
+
+```js
+var stub = sinon.stub(obj, "stubbedMethod").callsFake(function () {
+    return 42;
+});
+```
+
+Codemod available at https://github.com/hurrymaplelad/sinon-codemod
+
+## `sinon.stub(object, property, value)` - Removed
+The three parameter signature will now throw an exception (feature has been deprecated at `sinon@2.0.0` and has been finally removed from the codebase).
```

Comment:
Nitpicking: In JavaScript, they're called `arguments`, so this should read something like this:

> Calling `sinon.stub` with three arguments will throw an `Error`. This was deprecated with `sinon@2` and has been removed with `sinon@3`